src/hott/02-homotopies.rzk.md

```diff
@@ -164,7 +164,7 @@ It is sometimes useful to have this in inverse form.
   ( H : homotopy A B f g)
   ( x y : A)
   ( p : x = y)
-  : ( concat B (f x) (g x) (g y) (H x) (ap A B x y g p)) =
+  : (concat B (f x) (g x) (g y) (H x) (ap A B x y g p)) =
```

Comment:
This space should not be automatically removed since it is only preceded by single-character tokens.

Reply:
Perhaps the style guide should clarify this point further as it is not easy to check for the "opening parentheses marking a branch" with just lexical analysis. I was confused about this point as I saw examples of both styles, such as:
```rzk
Σ ( is-segal-A : is-segal A)
  , ( (x : A)
```
Note that all tokens before `x` are single-character, yet there is no space before it. Same with `( (f , g) : iff A B)`, for example.

Reply:
While working on this, I also noticed that the rule as you described it could also lead to such output:
```rzk
  ( C : ( x : A) → (y : B) → U)
  ( f : ( x : A) → (y : B) → C x y)
```
But I don't think it is desirable to add the space before the `x`, right?
I think this rule needs a more precise specification to reliably implement it in code

Reply:
Apologies, my description was not entirely accurate. I agree with you.

Reply:
No worries, just let me know once a stricter definition is agreed upon.
In the meantime, if all else is good, should I just disable this rule (temporarily) in the formatter, or did you find other problems? (Or should I wait to implement this rule correctly before merging?)

Reply:
Sorry, I didn't see this question until now. Let me think quickly

Reply:
Fixed by checking if the previous tokens consist solely of the following: `["(", ":", ",", "=", "→", "↦", "≡", "≤", "∧", "∨", "Σ", "Σ", "*₁", "0₂", "1₂", "×"]` (almost any built-in single-character token)

Reply:
Oh, that seems reasonable